### ISSUE TITLE
Question about logits from OpenAI models

### ISSUE THREAD

**Issue by jiangjiechen** [swj0419/detect-pretrain-code#2]
Hi Weijia,

Thank you for this interesting work! Just a very technical question: how do you get the bottom-k outlier logits from OpenAI models (i.e., davinci series)? It seems the GPT-3 perplexity calculation in your code only handles top-5 tokens from OpenAI API?

**Comment by swj0419** [comment]
Hi Jiangjie,

Thanks for your interest in our work. The min-k outlier logits refer to the low prob tokens in the input sentence. For example, "The Kremlin and local Russian leaders attempted Monday", the min-k outlier tokens could be "Kremlin, Russian ...".

**Comment by jiangjiechen** [comment]
Ah, I see! Thank you for the clarification! Great work! :D
